R/PythonPkgWrapperUtils.R

```diff
@@ -0,0 +1,443 @@
+# ------------------------------------------------------------------------------
+#
+#   Helpers for wrapping python packages
+#
+# ------------------------------------------------------------------------------
+
+#' Determines args and kwargs
+#'
+#' This function determines args and kwargs for a given list of values.
+#'
+#' @param ... the input values
+#' @return A list of args and kwargs
```

Comment:
The grouping of arguments into 'args' (the unnamed or positional arguments) and 'kwargs' (the named or keyword arguments) to be passed to the corresponding Python function.